unicore/ask/service/models.py

```diff
@@ -34,13 +35,22 @@ class QuestionResponse(Base, UUIDMixin):
     question_option_id = Column(
         UUIDType(binary=False), ForeignKey('question_options.uuid'),
         nullable=False)
+    created_at = Column(DateTime, default=datetime.utcnow())
```

Comment:
I forgot to mention, if you update the default like in the link above, you also have to update the migrations:
https://github.com/universalcore/unicore.hub/blob/develop/unicore/hub/service/alembic/versions/53661742b286_replace_app_password_with_key_add_url_.py#L50